src/GeneralTools/DataverseClient/Client/Microsoft.PowerPlatform.Dataverse.Client.csproj

```diff
@@ -37,13 +37,12 @@
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.7.0" />
     <PackageReference Include="System.Security.Cryptography.Algorithms" Version="4.3.1" />
     <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="4.7.0" />
-    <PackageReference Include="System.Security.Permissions" Version="5.0.0" />
-    <PackageReference Include="System.ServiceModel.Http" Version="4.8.1" />
```

Comment:
This is used by the WCF subsystem, and is a base requirement of Microsoft.Xrm.Sdk

Reply:
I think you're right. It's been a while but if I remember correctly, these packages were referenced transitively so what I meant, to be more precise, is that they do not need to be declared as top level dependencies. 

I don't insist on removing these references. I can edit my PR and leave out this part if you see value in rest of the changes.